eng/scripts/Publish-MavenPackages.ps1

```diff
@@ -240,24 +240,34 @@ foreach ($packageDetail in $packageDetails) {
     }
 
     $attempt = 0
+    $success = $false;
```

Comment:
Should we try to capture the exit code on failure? I'm not sure if it will give us much insight but it might be interesting.

Reply:
We emit it:   
``` powershell
Write-Information "Release attempt $attempt exited with code $LASTEXITCODE"
```

Should we `exit $LASTEXITCODE`?